.github/workflows/fork-sync.yml

```
@@ -0,0 +1,19 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # once a day
+  workflow_dispatch: # on button click
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
```

Why don't we use the latest version? CUrrent latest is [v2.0.10](https://github.com/marketplace/actions/fork-sync).
```suggestion
      - uses: tgymnich/fork-sync
```

---

BTW could you please also check https://github.com/wei/pull and any other available options. `wei/pull` seems to be significantly more popular and widely adopted than `tgymnich/fork-sync`.